cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -226,3 +227,49 @@ message SegYSeismicResponse {
 message SearchFilesRequest {
     SearchSpec spec = 1;
 }
+
+message SearchJobStatusRequest {
+    message Filter {
+        message Id {
+            oneof id {
+                string job_id = 1;
+                string file_id = 2;
+                string file_name = 3;
+            }
+        }
+
+        oneof filter {
+            Id id = 1;
+            JobStatus status = 2;
+            string target_storage_tier_name = 4;
+            google.protobuf.Timestamp started_before = 5;
+            google.protobuf.Timestamp started_after = 6;
+            google.protobuf.Timestamp updated_before = 7;
+            google.protobuf.Timestamp updated_after = 8;
+        }
+    }
+
+    enum Order {
+        DESCENDING = 0;
+        ASCENDING = 1;
+    }
+
+    Order order_by_updated = 1; // Defaults to most recently updated first
```

Comment:
Maybe there isn't much demand for this, could be removed and just use the default

Reply:
I think I agree with this sentiment. It's easier to add this later than remove it. Not that it's a huge burden either, but we can simplify the SQL query...

Reply:
Will remove